licenserc.toml

```
@@ -0,0 +1,55 @@
+headerPath = "scripts/license-header.txt"
+
+excludes = [
+    # Bazel files
+    ".bazelignore",
+    ".bazelrc",
+    ".bazelversion",
+    "BUILD.bazel",
+    "MODULE.bazel.lock",
+    "WORKSPACE",
+
+    # Root license file is more verbose
+    "LICENSE",
+
+    # Ignore scripts that drive releases
+    "scripts/*",
+
+    # Configuration files
+    "*.conf",
+    "*.toml",
+    "*.yaml",
+    "*.yml",
+
+    # Documentation files
+    "*.md",
+
+    # Pkl generated files
+    'PklProject.deps.json',
+
+    # Test specification files
+    "tests/**/*.txt",
```

All `.txt` files can be skipped, right?

```suggestion
    "*.txt",
```